examples/skr/maa-test/README.md

```diff
@@ -0,0 +1,103 @@
+# NOTE This example is for the Kata based AKS preview
```

Comment:
It has `Kata-based` lower down. I think changing this one to follow would look good